delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -50,7 +50,32 @@ def init_connection(self, access_info):
         self.array_id = access_info.get('extra_attributes', {}). \
             get('array_id', None)
         if not self.array_id:
-            raise exception.InvalidInput('Input array_id is missing')
+            """ Unisphere  managing  only one array (embedded unisphere)
+             can be supported with out array_id in input """
+
+            # Todo : Remove this temporary connection
+            #  while removing PyU4V dependency
+            try:
+                # Initialise PyU4V connection to VMAX
+                temp_conn = PyU4V.U4VConn(
+                    u4v_version=SUPPORTED_VERSION,
+                    server_ip=self.rest_host,
+                    port=self.rest_port,
+                    verify=False,
+                    username=self.rest_username,
+                    password=self.rest_password)
+
+            except Exception as err:
+                msg = "Failed to connect to VMAX: {}".format(err)
+                LOG.error(msg)
+                raise exception.StorageBackendException(msg)
+            # Get all array list managed by this unisphere
+            array_list = temp_conn.common.get_array_list()
+            if len(array_list) != 1:
```

Comment:
1. what happens, if array_list is empty. Then ...line#78 will fail.
2. The exception message not clear to me(may be it's to me only). Please check once, if it is applicable.